packages/terra-functional-testing/src/config/utils/getConfigurationOptions.js

```diff
@@ -12,12 +12,16 @@ const getConfigurationOptions = (options) => {
     browsers,
     buildBranch,
     buildType,
+    buildUrl,
     disableSeleniumService,
     ignoreScreenshotMismatch,
```

Comment:
nit: can you move this to the proper alphabetical location since we're already updating the file?

+1 on maintain alphabetical order for all the other additions

Reply:
Reorderd here https://github.com/cerner/terra-toolkit/pull/755/commits/8cf3c9541ebb57fc411545d6df927c6e950b05b8